### ISSUE TITLE
WIthdrawing validator reward gave a small amount of EXP

### ISSUE THREAD

**Issue by Reecepbcups** [notional-labs/craft#97]
```
craftd q bank balances craft1xxqq26cv7hsuzapvmrrkfrhjan7tuafawnj4qq --height 13641
- amount: "9999972000"
  denom: ucraft


craftd tx distribution withdraw-rewards $(craftd keys show pbcups --bech val -a) --commission --from $(craftd keys show pbcups -a) --chain-id craft-v4 --fees 200ucraft

craftd q tx --type=hash 22D119D1BA4694941DBBD5004FFC7AFCDA644BF9613F634AC957FCC072718922

craftd q bank balances craft1xxqq26cv7hsuzapvmrrkfrhjan7tuafawnj4qq --height 13642
- amount: "10000426400"
  denom: ucraft
- amount: "11"
  denom: uexp
```

2 validators have been jailed, so maybe when withdrawing it gave some EXP from their slash?

**Comment by nguyenvuong1122000** [comment]
I think distribution module send coin for your account, not slashing problem
![Screenshot from 2022-04-01 20-40-34](https://user-images.githubusercontent.com/56973102/161276143-6f5bdd5f-13b0-45ee-8e9e-cc05c53f4aab.png)

**Comment by nguyenvuong1122000** [comment]
craft1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8xfhtvr is distribution module address

**Comment by Reecepbcups** [comment]
> craft1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8xfhtvr is distribution module address

In the genesis.json, distribution has a 0.05 (5%) community tax. Since bonding denom is uexp, could that tax have been given to distribute to people who have rewards? (I assumed the slashed 0.2exp * 5% tax rate is being distributed or something like that)

If so, we need some way to disable distribution from giving exp when getting commission / rewards. (Only way to get EXP is from that DAO proposal w/ your exp module)

**Comment by faddat** [comment]
nice work. 

For this one, we have to revisit after the rebuild.

The rebuild will be near total or total.

**Comment by Reecepbcups** [comment]
Seems to be fixed in v46.0
